packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -0,0 +1,129 @@
+{
+  "brand": {
+    "VideoPlayer": {
+      "border": {
+        "radius": {
+          "value": "var(--brand-borderRadius-medium)"
+        }
+      },
+      "controls": {
+        "gap": {
+          "value": "15px"
+        },
+        "background": {
+          "value": "rgba(0, 0, 0, 0.2)"
+        },
+        "backgroundGradient": {
+          "value": "linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6))"
+        },
+        "padding": {
+          "value": "var(--base-size-16) var(--base-size-24)"
+        },
+        "icon": {
+          "height": {
+            "value": "24px"
+          }
+        },
+        "focus": {
+          "borderColor": {
+            "value": "var(--brand-color-focus)"
+          },
+          "boxShadow": {
+            "value": "0 0 0 2px var(--brand-color-focus)"
+          }
+        }
+      },
+      "closedCaption": {
+        "padding": {
+          "vertical": {
+            "value": "1px"
+          },
+          "horizontal": {
+            "value": "5px"
+          }
+        },
+        "backgroundColor": {
+          "value": "var(--brand-color-white)"
+        },
+        "color": {
+          "value": "black"
+        },
```

Comment:
We don't use named colors, and prefer tokens. Could we please:

a) use our base color for black instead `var(--base-color-scale-black-0)`
b) move colors into their own JSON file, and provide the dark color mode value also (should be able to reference other color source files for examples of how we do this)